### ISSUE TITLE
Apply-Damage Ballistic Armor Not working

### ISSUE THREAD

**Issue by twinlock** [vkdolea/age-system#69]
See below: character is using a ballistic weapon but armor calculation isn't working correctly
<img width="423" alt="Screenshot 2021-10-24 102212" src="https://user-images.githubusercontent.com/17128304/138605962-f67820bf-d41d-4aba-856a-11ac06f5d490.png">

Tracked this issue down looks like there's a typo on [apply-damage.js#220](https://github.com/vkdolea/age-system/blob/main/modules/apply-damage.js#L220). 

Just need to replace `healthSys._useBallistic;` with `healthSys.useBallistic;` tested this locally, it works. 

**Sidenotes**:  Woulda made a PR, buuuuuuut couldn't figure that out. Maybe I'm a fool but whether it's permissions or my lack of github finesse, but I couldn't create a branch to make this a PR (keep gettin permission errors). Maybe add a contributor's guide? I don't have much js experience, but for things like this it'd be great to just help ya out

**Comment by vkdolea** [comment]
Good catch! This was a stupid mistake from my side... The next release will add this fix :-)

I am also not familiar with PR from Github as well... I'll learn it and add a contributor's guide, as you suggested. Cheers!
